### ISSUE TITLE
Animation flickering

### ISSUE THREAD

**Issue by crissNb** [FelixKratz/yabai#14]
As you can see from the video I've provided, there seem to be a flicker happening right when the animation ends, instead of a smooth fade transition. Am I missing something?

Using macOS Ventura 13.2.1 and yabairc from your dotfiles (so the WindowManager is disabled)

https://user-images.githubusercontent.com/83137305/219903354-d6b92588-23fc-412b-8618-0b66d22271a9.mov

**Comment by FelixKratz** [comment]
Try:
```bash
sudo yabai --uninstall-sa
sudo yabai --load-sa
brew services restart fyabai
```

**Comment by crissNb** [comment]
That did the trick. Thanks for the quick response!

**Comment by kinglouie** [comment]
@FelixKratz I have the same issue as the OP. At first I tried to install your fork by just using your homebrew formulae but with that the scripting addition did not work at all (on 13.6 m2). So I cherry picked the code for crossfade and the ultrawide padding stuff and now the scripting addition works but I got the same flickering issues as in the video above. Already tried your proposed fix to no avail.
Is there something else I could try?

This is the Fork i created https://github.com/kinglouie/yabai/commits/fork
